addons/mail/static/src/components/composer/composer.js

```diff
@@ -48,11 +48,24 @@ class Composer extends Component {
          * Reference of the text input component.
          */
         this._textInputRef = useRef('textInput');
+        /**
+         * Reference of the subject input. Useful to set content.
+         */
+        this._subjectRef = useRef('subject');
         this._onClickCaptureGlobal = this._onClickCaptureGlobal.bind(this);
     }
 
     mounted() {
         document.addEventListener('click', this._onClickCaptureGlobal, true);
+        if(this._subjectRef.el) {
+            this._subjectRef.el.value = this.composer.subjectContent;
+        }
+    }
+
+    patched() {
+        if(this._subjectRef.el) {
+            this._subjectRef.el.value = this.composer.subjectContent;
+        }
```

Comment:
Can't we remove this part now that we use t-att-value?

Reply:
Nop

![image](https://user-images.githubusercontent.com/48204996/85597900-59959880-b64b-11ea-8374-8d1dd8be33ab.png)

Reply:
@jgi-odoo had this same issue when working on `ComposerTextInput`, and had to keep assigning `.value` in `mounted()/patched()` as a result.

Reply:
Ok but then `t-att-value` in template seems useless since the value is always assigned/overridden in JS.

Reply:
I may be wrong @seb-odoo but didn't we find out ```t-att-value``` works as default value ?